other_projects_that_use_sympy.tex

```diff
@@ -0,0 +1,69 @@
+There are several projects that use SymPy as a library for implementing
+a part of their project, or even as a part of back-end for their 
+application as well.
+
+Some of them are listed below-:
+
+\begin{itemize}
+\item
+  \href{http://cadabra.science/index.html}{\textbf{Cadabra}}: Cadabra is
+  a symbolic computer algebra system (CAS) designed specifically for the
+  solution of problems encountered in field theory.
+\item
+  \href{http://octave.sourceforge.net/symbolic/}{\textbf{Octave Symbolic}}:
+  The Octave-Forge Symbolic package adds symbolic calculation features
+  to GNU Octave. These include common Computer Algebra System tools such
+  as algebraic operations, calculus, equation solving, Fourier and
+  Laplace transforms, variable precision arithmetic and other features.
+\item
+  \href{https://github.com/jverzani/SymPy.jl}{\textbf{SymPy.jl}}:
+  Provides a Julia interface to SymPy using PyCall.
+\item
+  \href{https://mathics.github.io/}{\textbf{Mathics}}: Mathics is a
```

Comment:
Latex warning here:
Overfull \hbox (25.15338pt too wide) in paragraph at lines 22--27